TF_Settings_Web/src/Components/Pages/Camera/QuickSetup/Calibration/CalibrationScreen.tsx

```diff
@@ -0,0 +1,145 @@
+/* eslint-disable @typescript-eslint/no-empty-function */
+import 'Styles/Camera/Calibrate.scss';
+import styles from 'Styles/Camera/Calibrate.scss';
+
+import React, { useEffect } from 'react';
+import { useLocation, useNavigate } from 'react-router-dom';
+
+import { ConnectionManager } from 'TouchFree/Connection/ConnectionManager';
+import { InputActionManager } from 'TouchFree/Plugins/InputActionManager';
+import { InputType, InteractionType, TouchFreeInputAction } from 'TouchFree/TouchFreeToolingTypes';
+
+import {
+    CalibrationHandLostMessage,
+    CalibrationCancelButton,
+    CalibrationInstructions,
+    CalibrationTutorialVideo,
+    CalibrationProgressCircle,
+} from './CalibrationComponents';
+
+const { handNotFoundHeight } = styles;
+
+interface CalibrationScreenProps {
+    isHandPresent: boolean;
+    onCancel: () => void;
+}
+
+export const TIMEOUT_S = 10;
+const TIMEOUT_MS = TIMEOUT_S * 1000;
+const INITIAL_WAIT_MS = 5000;
+
+const CalibrationScreen: React.FC<CalibrationScreenProps> = ({ isHandPresent, onCancel }) => {
+    const [displayHandIndicator, setDisplayHandIndicator] = React.useState(false);
+    // ===== Click Progress =====
+    const [progressToClick, setProgressToClick] = React.useState(0);
+    const [progress, setProgress] = React.useState(0);
+    const isNewClick = React.useRef(false);
+    // ===== Timeout =====
+    const [timeToPosSelect, setTimeToPosSelect] = React.useState(TIMEOUT_S);
+    const timeout = React.useRef<NodeJS.Timeout>();
```

Comment:
Is it weird it is using NodeJS.Timeout?
If you use window.setTimeout it returns a number or could use `ReturnType<typeof setTimeout>`